src/_gettsim/transfers/rente.py

```diff
@@ -19,8 +19,181 @@ def sum_ges_rente_priv_rente_m(priv_rente_m: float, ges_rente_m: float) -> float
     return out
 
 
-@policy_info(params_key_for_rounding="ges_rente")
-def ges_rente_vor_grundr_m(
+@policy_info(
+    start_date="2021-01-01",
+    params_key_for_rounding="ges_rente",
+    name_in_dag="ges_rente_m",
+)
+def ges_rente_mit_grundrente_m(
+    bruttorente_m: float,
+    grundr_zuschlag_m: float,
+    rentner: bool,
+) -> float:
+    """Calculate total individual public pension including Grundrentenzuschlag. Is only
+    active after 2021 when Grundrente is in place.
+
+    Parameters
+    ----------
+    bruttorente_m
+        See :func:`bruttorente_m`.
+    grundr_zuschlag_m
+        See :func:`grundr_zuschlag_m`.
+    rentner
+        See basic input variable :ref:`rentner <rentner>`.
+
+    Returns
+    -------
+
+    """
+    # Return 0 if person not yet retired
+    out = bruttorente_m + grundr_zuschlag_m if rentner else 0.0
+    return out
+
+
+@policy_info(end_date="2020-12-31")
+def ges_rente_m(bruttorente_m: float) -> float:
+    return bruttorente_m
+
+
+@policy_info(
```

Comment:
Move down (at least I thought that typically we order things as time progresses, not latest-on-top).